korangar/src/graphics/renderers/picker/target.rs

```diff
@@ -5,32 +5,29 @@ use ragnarok_packets::EntityId;
 #[cfg(feature = "debug")]
 use crate::world::MarkerIdentifier;
 
-#[cfg(feature = "debug")]
-enum MarkerEncoding {
-    Object = 10,
-    LightSource = 11,
-    SoundSource = 12,
-    EffectSource = 13,
-    Entity = 14,
-    Shadow = 15,
+#[repr(u32)]
+enum PickerValueType {
+    Tile = 1,
+    Entity = 2,
+    #[cfg(feature = "debug")]
+    ObjectMarker = !0,
+    #[cfg(feature = "debug")]
+    LightSourceMarker = !0 - 1,
+    #[cfg(feature = "debug")]
+    SoundSourceMarker = !0 - 2,
+    #[cfg(feature = "debug")]
+    EffectSourceMarker = !0 - 3,
+    #[cfg(feature = "debug")]
+    EntityMarker = !0 - 4,
+    #[cfg(feature = "debug")]
+    ShadowMarker = !0 - 5,
```

Comment:
Just out of curiosity, why did you decide to fill the `u32` with `1`s for the markers?

I think at this point I would set `Tile = 1` and let the other enum values be inferred.

Reply:
Are there other places than the tests (and the tile shader that we want to use specialization constants for now)?

If it's only the tests I would prefer having to fix them over giving the variants arbitrary values. On that note, we can also start at `0` now right? So we don't even need `Tile = 1`

Reply:
I think the tests are the last place. If we make Tile = 0, what would the clear value? I mean, we need something that means "nothing selected".

(High 0, Low 0) would then mean "Tile at X=0, Y=0", right?

Reply:
We check those implicitly by using it as a fall through case. But in the old branch, we didn't even gave the clear value a linked enum value. The current branch does this now. So if a value inside the picker texture has the clear value or any other unknown type value, it will be interpreted as "Nothing".